### PR TITLE
feat: room6 New graphics

### PR DESCRIPTION
**Pull request by balloonpopper**
This change is 95% complete. There's an outstanding issue
See https://github.com/godot-escoria/escoria-issues/issues/177

I'll need to remove the hack (room6.esc lines 14 and 15) that immediately unlock the second door (to allow the bug to be investigated) before the room is production-ready.

image

### CODE REVIEW

**Review by StraToN — APPROVED** [review]

**Comment by balloonpopper** [comment]
Currently broken pending issue 193 for set_state command

**Comment by balloonpopper** [comment]
@StraToN I'm happy with however you want the commit history to look. If you want to push #554 separately for the commit history, I'll go back a commit and rebase. Otherwise we'll just merge this one. Whichever  you like :-)

**Comment by StraToN** [comment]
> 

I'm alright that way :)

**Comment by StraToN** [comment]
@balloonpopper Please remove commit 0f8d53e8e956cfd593bf2968e99ccfa3d0a4ecff from your PR. I fear it will collide with changes on this brought by #554

**Review by StraToN — CHANGES_REQUESTED** [review]
Please remove commit 0f8d53e8e956cfd593bf2968e99ccfa3d0a4ecff. Thanks!

**Comment by balloonpopper** [comment]
Already done @StraToN - the only files in the PR now are room06 ones.

File: game/rooms/room06/esc/worker.esc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :look
-say player "He's looking a bit odd."
+say player "He's wearing a badge that says "Careers Officer"."
```
Comment (BHSDuncan) [inline game/rooms/room06/esc/worker.esc:2]: Do the quotation marks around "Careers Officer" work all right? Do they need escaping?

Comment (balloonpopper) [reply game/rooms/room06/esc/worker.esc:2]: They don't need escaping.
image

Comment (BHSDuncan) [reply game/rooms/room06/esc/worker.esc:2]: #unexpectedparsingwin

**Review by BHSDuncan — APPROVED** [review]

**Review by StraToN — APPROVED** [review]

**Pull request merged**